legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java

```diff
@@ -13,10 +13,12 @@
 	private Long id;
 	private LegacyArrowheadService providedService;
 	private LegacyArrowheadSystem provider;
-	private String serviceUri;
+	private String serviceURI;
```

Comment:
Regarding this as we have investigated now 4.1.2 client skeleton has "serviceUri" filed, but indeed the 4.1.2 FW has "serviceURI". The reason why it cloud work is that 4.1.2 FW object parser is not case sensitive. As I just tested out we can solve this issue with having both type of getter and setter methods, therefore the object parser of translater is able to handle both type of request.

Reply:
I've pushed this update to the master branch.